src/test/java/com/fasterxml/jackson/dataformat/xml/XmlMapperTest.java

```diff
@@ -22,11 +20,7 @@ public void smokes() throws Throwable {
     }
 
     private static void _smokes(JenkinsRule r) throws Throwable {
-        XMLInputFactory inputFactory =
-                XMLInputFactory.newFactory(
-                        XMLInputFactory.class.getName(), XmlFactory.class.getClassLoader());
-        XmlFactory factory = new XmlFactory(inputFactory);
-        XmlMapper mapper = new XmlMapper(factory);
```

Comment:
Why? Was this change required or optional?

Reply:
it was a workaround for a classloading issue in jackson < 2.12.4 (classloading issue still present till jackson 2.13.0 but jackson has included a workaround now)

see https://github.com/jenkinsci/jackson2-api-plugin/pull/82 https://github.com/FasterXML/jackson-dataformat-xml/issues/483 https://github.com/FasterXML/jackson-dataformat-xml/pull/480